licenses/cgmanifest.json

```diff
@@ -66,6 +56,42 @@
       },
       "DevelopmentDependency": false
     },
+    {
+      "Component": {
+        "Type": "linux",
+        "linux": {
+          "Name": "libcurl4",
+          "Version": "7.58.0-2ubuntu3.14",
+          "Distribution": "Ubuntu",
+          "Release": "18.04"
+        }
+      },
+      "DevelopmentDependency": false
+    },
+    {
+      "Component": {
+        "Type": "linux",
+        "linux": {
+          "Name": "libcurl4",
+          "Version": "7.52.1-5+deb9u15",
+          "Distribution": "Debian",
+          "Release": "9.13"
+        }
+      },
+      "DevelopmentDependency": false
+    },
+    {
+      "Component": {
+        "Type": "linux",
+        "linux": {
+          "Name": "libcurl4",
+          "Version": "7.64.0-4+deb10u2",
+          "Distribution": "Debian",
+          "Release": "10.10"
+        }
+      },
+      "DevelopmentDependency": false
+    },
     {
       "Component": {
```

Comment:
just wondering - do we need to also include the threads library here?

Reply:
No idea. Is it considered a 3rd party lib? We don't list c/c++ libs here either.